### PR TITLE
Chore refactor report detail view

### PR DESCRIPTION
**Pull request by Szymiks**
Resolves: https://github.com/Code-Poets/sheetstorm/issues/245
**Done**
- update should work
- template should display and allow to send post only for users belonging to given projects
- django view

### CODE REVIEW

File: employees/tests/test_unit_report_viewset.py [hunk 1]
```diff
@@ -430,130 +433,75 @@ def test_serializer_project_field_queryset_should_contain_only_projects_user_is_
...
-        self.user = CustomUser(
-            email="testuser@codepoets.it", password="newuserpasswd", first_name="John", last_name="Doe", country="PL"
-        )
-        self.user.full_clean()
-        self.user.save()
-
-        self.project = Project(name="Test Project", start_date=datetime.datetime.now())
-        self.project.full_clean()
-        self.project.save()
+        super().setUp()
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_viewset.py]: After refactoring to django view please move tests related to this view to `test_views.py`.

File: employees/templates/employees/report_detail.html [hunk 2]
```diff
@@ -13,15 +13,14 @@
...
-        {% csrf_token %}
-        {% render_form serializer template_pack='rest_framework/vertical' %}
-        <input type="submit" value="{{ UI_text.UPDATE_REPORT_BUTTON.value }}">
-        <input type="submit" name="discard" value="{{ UI_text.DISCARD_CHANGES_BUTTON.value }}">
-        <input type="button" id="opener" value="{{ UI_text.DELETE_REPORT_BUTTON.value }}">
+       <div class="modal-dialog">
+            {% csrf_token %}
+            {{ form|crispy }}
+            <input type="submit" value="{{ UI_text.UPDATE_REPORT_BUTTON.value }}">
+           <button type="button" name="discard"><a href="{% url 'custom-report-list' %}">{{ UI_text.DISCARD_CHANGES_BUTTON.value }}</a></button>
```
Comment (rwrzesien) [inline employees/templates/employees/report_detail.html]: Please align.

File: employees/tests/test_unit_report_viewset.py (hunk 1 above)
Comment (Szymiks) [reply employees/tests/test_unit_report_viewset.py]: Done

File: employees/templates/employees/report_detail.html (hunk 2 above)
Comment (Szymiks) [reply employees/templates/employees/report_detail.html]: Done

**Pull request merged**